components/service/experiments/proguard-rules.pro

```diff
@@ -19,3 +19,6 @@
 # If you keep the line number information, uncomment this to
 # hide the original source file name.
 #-renamesourcefileattribute SourceFile
+
+# Experiments specific protections
+-keep class mozilla.components.service.experiments.** { *; }
```

Comment:
What's the intent of this? If you want users of the library to retain this, you need to add this rule to a `proguard-rules-consumer.pro` (see [this example](https://searchfox.org/glean/source/glean-core/android/proguard-rules-consumer.pro)) and then advertise it in your [gradle file](https://searchfox.org/glean/rev/c6e1349fdb05baec59aaa5f465ba65e6cdd3b7d3/glean-core/android/build.gradle#53,59).

Reply:
Ahh, thanks for this!  I wasn't involved with messing with proguard/R8 rules when we were having problems with it so I guess I didn't realize that they needed to be implemented this way for consuming apps.  Let me change it :)